cmd/kube-dns/app/server.go

```diff
@@ -163,40 +162,6 @@ func setupSignalHandlers() {
 	}()
 }
 
-func validateHostAndPort(hostAndPort string) error {
```

Comment:
I think this should just go into config/ package instead of creating a new package `util`

Reply:
It's now in config package.
I am making it public for now in testing purpose. Will change it to private method very soon on Monday.

Reply:
Two validation logic merged. Now skydns's nameserver is synced/guarded by kubedns.